apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V66__new_schema_snf_claims_fix.sql

```diff
@@ -0,0 +1,522 @@
+-- NEW_SCHEMA_SNF_CLAIMS_FIX.SQL
+-- flyway migration for SNF_CLAIMS_NEW and SNF_CLAIM_LINES_NEW tables into
+-- a new schema structure; previus flyway script, V63__nw_schema_snf_claims.sql
+-- whcich erroneously defined columns in the snf_claim_lines_new table
+-- as an integer. This script fixes that mistake and does a complete data
+-- migration of SNF claims data because the _new tables would have missed
+-- a weekend's ETL propcessing.
```

Comment:
There are some spelling errors in this comment that should be corrected.

Reply:
fixed. thx.